### PR TITLE
add exponential_buckets for APPLY_PROPOSAL metrics

### PR DESCRIPTION
**Pull request by javaforfun**

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay

File: src/raftstore/store/worker/metrics.rs [hunk 1]
```diff
@@ -80,6 +80,7 @@ lazy_static! {
     pub static ref APPLY_PROPOSAL: Histogram =
         register_histogram!(
             "tikv_raftstore_apply_proposal",
-            "Proposal count of every region in a mio tick"
+            "Proposal count of all region in a mio tick",
```
Comment (BusyJay) [inline src/raftstore/store/worker/metrics.rs]: regions?

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -907,7 +907,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
         self.raft_metrics.ready.pending_region += pending_count as u64;
 
-        let mut region_proposals = Vec::with_capacity(pending_count);
+        let mut region_proposals = vec![];
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: please use another PR.

**Pull request merged**